packages/core/src/errors.ts

```diff
@@ -48,3 +48,14 @@ export class InvalidStringError extends CustomError {
     super(`Invalid string: "${expectation}"`);
   }
 }
+
+export enum CardanoNodeFailure {
```

Comment:
See https://github.com/input-output-hk/cardano-js-sdk/blob/0.3.0/packages/core/src/Cardano/types/TxSubmissionErrors.ts for reference of the existing pattern used for inheriting the Ogmios errors.

These are the list of errors we'll need to include
https://github.com/CardanoSolutions/ogmios/tree/master/clients/TypeScript/packages/client/src/errors